Rebuild-Black-Business/RBB-Website
# Add Fathom analytics

This will add Fathom for analytics per [this issue](https://trello.com/c/AZccLvQH/13-add-analytics-google).

# To test:
1. set an environment variable called `FATHOM_SITE_ID` with the appropriate ID from fathom to test. This can be done in a `.env.development` file if you're running `npm start` locally, or `.env.production` if you're running a full build.  (On netlify/github pages/etc it'll have to be set once this is merged)
2. Start the site, and inspect it - you should see a `<script>` tag just before `</head>` pointing to usefathom, with the ID you furnished:

![image](https://user-images.githubusercontent.com/1844496/83707466-0a2df080-a5e8-11ea-885d-f658bb4fc735.png)

3. (...once fathom access is set up), verify page views on fathom's dashboard
